languages/common-lisp/exercises/concept/arithmetic/.docs/after.md

```diff
@@ -0,0 +1,114 @@
+## Summary
+
+The goal of this exercise was to get comfortable with prefix-arithmetic in
+Common Lisp and learn about integer and floating-point numbers.
+
+In addition to the traditional addition, subtraction, multiplication, and
+division operations, you encountered different rounding functions as well as
+functions that take the square-root or raise some number to a given power.
+
+With these tools, you should be well equipped to handle arithmetic problems in
+Common Lisp.
```

Comment:
This section refers to the contents of the exercise (and implicitly the `introduction.md` document). The `after.md` is considered to be a stand-alone document that students can revisit later, as a sort of reference documentation for the concepts involved. As such, all the information that is in the `introduction.md` should also be in the `after.md` document.